backend/api.js

```diff
@@ -0,0 +1,135 @@
+const express = require('express')
+const app = express()
+const port = 3001;
+const cors = require('cors')
+const parser = require('body-parser')
+const knex = require('knex')(require('./knexfile.js')['development']);
+//cors policy update
+app.use(cors())
+app.use(parser.json())
```

Comment:
It's generally best practice to also use parser.urlencoded({extended:true}) here.  This gives us more flexibility on the client or server end in how we compose and handle requests respectively.  [Read more about that here](http://expressjs.com/en/resources/middleware/body-parser.html) or [read about the express-integrated version here](http://expressjs.com/en/api.html#express.urlencoded). 

Note that express also now provides express.json(), which is based on body-parser's functionality, as is express.urlencoded({}).  These were integrated directly into express due to being so widely used, which will allow you to avoid having to install additional dependencies.